src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/CaseCommentResponse.java

```diff
@@ -24,17 +24,19 @@ public class CaseCommentResponse {
     private final String author;
     private final String createdByUuid;
     private final boolean draft;
+    private final boolean legacy;
 
     public static CaseCommentResponse of(CaseCommentEntity caseCommentEntity) {
         return CaseCommentResponse.builder()
-            .comment(caseCommentEntity.getComment())
-            .commentId(caseCommentEntity.getId())
-            .caseId(caseCommentEntity.getCaseId())
-            .defendantId(caseCommentEntity.getDefendantId())
-            .created(caseCommentEntity.getCreated())
-            .author(caseCommentEntity.getAuthor())
-            .createdByUuid(caseCommentEntity.getCreatedByUuid())
-            .draft(caseCommentEntity.isDraft())
-            .build();
+                .comment(caseCommentEntity.getComment())
+                .commentId(caseCommentEntity.getId())
+                .caseId(caseCommentEntity.getCaseId())
+                .defendantId(caseCommentEntity.getDefendantId())
+                .created(caseCommentEntity.getCreated())
+                .author(caseCommentEntity.getAuthor())
+                .createdByUuid(caseCommentEntity.getCreatedByUuid())
+                .draft(caseCommentEntity.isDraft())
+                .legacy(caseCommentEntity.isLegacy())
```

Comment:
I think if you want to use `isLegacy()` you'd need to change the column name to `IS_LEGACY` in the `CaseCommentEntity.java`

like the way it does for "IS_DRAFT"
```
 @Column(name = "IS_DRAFT", nullable = false)
    private boolean draft;
 ```
 
 or you can use `caseCommentEntity.getLegacy()` i think

Reply:
It autocompletes to isLegacy based on the boolean

Another example here - 

![Screenshot 2024-03-27 at 15 59 03](https://github.com/ministryofjustice/court-case-service/assets/5918563/e953288a-184f-41ba-8472-2fd93b62b44f)